debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/structureformat/extensions/J9ModuleStructureFormatter.java

```diff
@@ -0,0 +1,71 @@
+/*******************************************************************************
+ * Copyright (c) 2018, 2018 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package com.ibm.j9ddr.vm29.tools.ddrinteractive.structureformat.extensions;
+
+import java.io.PrintStream;
+import java.util.List;
+
+import com.ibm.j9ddr.CorruptDataException;
+import com.ibm.j9ddr.tools.ddrinteractive.BaseStructureFormatter;
+import com.ibm.j9ddr.tools.ddrinteractive.Context;
+import com.ibm.j9ddr.tools.ddrinteractive.FormatWalkResult;
+import com.ibm.j9ddr.tools.ddrinteractive.IFieldFormatter;
+import com.ibm.j9ddr.vm29.pointer.generated.J9ModulePointer;
+import com.ibm.j9ddr.vm29.pointer.helper.J9ObjectHelper;
+
+/**
+ * Structure Formatter that adds a suffix like this:
+ * 
+ * Module name: java.instrument
+ * To display all j9packages exported by the module, use !dumppackageexports 0x000001305F46DB38
+ * To display all modules that the target module reads, use !dumpmodulereads 0x000001305F46DB38
+ * 
+ * When a !j9module is formatted
```

Comment:
Should there more to this sentence/paragraph?

Reply:
There isn't.. I used [J9ClassStructureFormatter.java](https://github.com/Ailloviee/openj9/blob/DDRCommands_v3/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/structureformat/extensions/J9ClassStructureFormatter.java) as template for this one. If you wish, I can make the comments more specified to both files. (The suffix shows up when we run `!j9module $moduleAddress$`)

Reply:
Please do fix the comment in this file. Poor comments in other files can be dealt with separately.